CorsixTH/Lua/entities/humanoids/vip.lua

```diff
@@ -516,7 +516,7 @@ function Vip:afterLoad(old, new)
     -- VIP's room visit chance is 50% (1/2) if total rooms in hospital is less than 80, else decided by total rooms / 40 (1/3, 1/4 etc)
     local rooms_threshold = 79
     if #self.world.rooms > rooms_threshold then
-      vip.room_visit_chance = math.floor(#self.world.rooms / 40)
+      self.room_visit_chance = math.floor(#self.world.rooms / 40)
     else
       self.room_visit_chance = 1
```

Comment:
How is `1` equal to `50%`?

BTW: The simpler solution to computing this is `self.room_visit_chance = math.max(1, math.floor(#self.world.rooms / 40))`